src/myr/Myr.js

```diff
@@ -421,6 +430,26 @@ class Myr {
         return this.cursor.color;
     }
 
+    setTexture = (texture = "") => {
+        let textures = TexturePack();
+        let textureTitle = [...textures.TexturePack.map(obj => obj.title)];
+        let textureURL = [...textures.TexturePack.map(obj => obj.url)];
+
+        this.cursor.texture = textureURL[textureTitle.indexOf(texture)];
+
+        let urlregex = /^(https?|ftp):\/\/([a-zA-Z0-9.-]+(:[a-zA-Z0-9.&%$-]+)*@)*((25[0-5]|2[0-4][0-9]|1[0-9]{2}|[1-9][0-9]?)(\.(25[0-5]|2[0-4][0-9]|1[0-9]{2}|[1-9]?[0-9])){3}|([a-zA-Z0-9-]+\.)*[a-zA-Z0-9-]+\.(com|edu|gov|int|mil|net|org|biz|arpa|info|name|pro|aero|coop|museum|[a-zA-Z]{2}))(:[0-9]+)*(\/($|[a-zA-Z0-9.,?'\\+&%$#=~_-]+))*$/;
+        if(urlregex.test(texture) || (texture === "")) {
+            this.cursor.texture = texture;
+        }
```

Comment:
What would happen if the URL fails the regex?  Does it notify the user in some way?

Reply:
No texture is displayed. The user currently isn't notified, I'm adding a console.error statement. Two probably best solutions would either be an error popup or and error texture however just because something passes the regex doesn't mean a texture will be read/displayed as it can get blocked by the CORS policy when the scene is run and tries to load the URL. So there would be some confusion as to why sometimes an error is shown and sometimes one is not despite neither working.